Cargo.toml

```diff
@@ -8,3 +8,4 @@ description = "A fun language with no classes whatsoever"
 clap = "^2.34"
 maplit = "^1.0"
 rustyline = "^9.1"
+rand = "^0.8"
```

Comment:
Please can you put these in alphabetical order? Sorry it's just annoying me a tiny bit lol

Reply:
Ok, will do

Reply:
Done!